acceptance/config/beaker/jenkins/el-6-x86_64-mdca.cfg

```diff
@@ -8,6 +8,13 @@ HOSTS:
     platform: el-6-x86_64
     template: Delivery/Quality Assurance/Templates/vCloud/centos-6-x86_64
     hypervisor: vcloud
+    puppetbin: /opt/puppetlabs/agent/bin/puppet
+    puppetbindir: /opt/puppetlabs/agent/bin
+    puppetvardir: /opt/puppetlabs/agent/cache
+    puppetpath: /etc/puppetlabs/agent
+    facterbindir: /opt/puppetlabs/agent/bin
+    hierabindir: /opt/puppetlabs/agent/bin
+
```

Comment:
Okay apologies for my confusing barrage of comments, but given the availability of these same values in Beaker 2.2.0 and later, I would suggest using those instead of setting project-specific values anywhere in the puppet-server repo.